code/config/resCodes.js

```diff
@@ -2,6 +2,7 @@ const authCodes = require('./resCodes/user/auth')
 const tripCodes = require('./resCodes/user/trip')
 const donationCodes = require('./resCodes/user/donation')
 const profileCodes = require('./resCodes/user/profile')
+const countryCodes = require('./resCodes/country_page/country')
```

Comment:
Could you change the name of the res code folder for country_page to countryPage to match the other folders ( the only one that is not like this is node_mudules and that is because we don't define that folder name )

Reply:
Done. Thanks!